main/main.cpp

```diff
@@ -1810,7 +1817,16 @@ bool Main::iteration() {
 		target_ticks = MIN(MAX(target_ticks, current_ticks - time_step), current_ticks + time_step);
 	}
 
-	return exit;
+#ifdef TOOLS_ENABLED
+	if (auto_build_solutions) {
+		auto_build_solutions = false;
+		if (!EditorNode::get_singleton()->call_build()) {
+			ERR_FAIL_V(true);
+		}
+	}
+#endif
+
+	return exit | auto_quit;
```

Comment:
Shouldn't it be `||`?

Reply:
Yep, I'm not sure why I did that. Should be logical instead of bitwise now :)